### DISCUSSION TITLE
Using LaTeX asterisk inside markdown

### DISCUSSION THREAD

**Discussion by nopria** [Witiko/markdown#278] [Q&A] [answered]
I need to use asterisks (like in `\begin{figure*}`) within an `hybrid` markdown environment, how can it be accomplished?

Of course if I place the asterisks as needed by LaTeX, ignoring that I'm inside a markdown environment, they are misinterpreted and I get the error

`! Missing \endcsname inserted.`

**Comment by Witiko** [comment, marked as answer]
@nopria In Markdown, asterisks denote [emphasis][1]. To prevent this interpretation, you have several options:

1. Escape  asterisks that do not denote emphasis: ```\begin{figure\*}```
2. Enable [the `rawAttribute` option][2] and surround TeX code in a raw code span: ``` `\begin{figure}`{=tex} ```
    Alternatively, you can also enable the `fencedCode` option and use a raw block:
    ````` tex
    ``` {=tex}
    \begin{figure}
    ```
    ``````
   Unlike `hybrid`, `rawAttribute` allows you to cleanly separate TeX and Markdown code.
   If you use TeX sparingly, disabling `hybrid` and using just `rawAttribute` will improve readability.

 [1]: https://daringfireball.net/projects/markdown/syntax#em
 [2]: https://witiko.github.io/markdown/#option-rawattribute
